### DISCUSSION TITLE
Ocpp 1.6 with twc gen 3

### DISCUSSION THREAD

**Discussion by alexeiw123** [ngardiner/TWCManager#592] [General]
Hi @MikeBishop and @ngardiner.

Came across a post on FB that led to this: https://www.energymining.sa.gov.au/__data/assets/pdf_file/0005/1051358/27062024-EVSE-Registered-Equipment-Compliance-List.pdf

It seems that TWC gen 3 is listed as OCPP 1.6 compliant.
This image also shows the API returning this as confirmation. No idea how to access or interact with this functionality.

Could this be a pathway to local start/stop and amperage control in gen 3?

![FB_IMG_1720835213668](https://github.com/user-attachments/assets/c3536d2a-6ef8-497d-ab8f-ae98e16ded10)

**Comment by RichieB2B** [comment]
While in theory it is possible to control amperage via OCPP I'm not sure this will help us. Even EVCC which supports OCPP has the TWC v3 [listed](https://docs.evcc.io/en/docs/devices/chargers#tesla-wall-connector-gen-3) as:

> The TWC wallbox cannot be controlled directly. Control is via the vehicle. The vehicle must be selected at the TWC3 loadpoint.

One reason could be that the TWC v3 does not allow to set arbitrary an OCPP URI.
